src/components/TodoCheckbox/TodoCheckbox.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { TickIcon, CrossIcon, IconsContainer } from "./TodoCheckbox.styled";
+import { ChatIcon } from "./TodoCheckbox.styled";
+
+function TodoCheckbox({
+  checkStatus,
+}: {
+  checkStatus: "unchecked" | "checked" | "crossed";
```

Comment:
this is repeated everywhere.

We should DRY https://topdev.vn/blog/yagni-dry-la-gi-nguyen-tac-yagni-dry-trong-java/

Create a custom type for checkStatus and reuse